### PR TITLE
Update task.md

### PR DESCRIPTION
**Pull request by stephen-hero**
language checked

### CODE REVIEW

File: Coroutines/Callbacks/task.md [hunk 1]
```diff
@@ -36,14 +36,14 @@ which uses callbacks instead of blocking calls.
     ```
 
    By calling `SwingUtilities.invokeLater`, you ensure that the `updateResults()` call, which updates the results,
-   happens on the main UI thread (AWT event dispatching thread).
+   occurs on the main UI thread (AWT event dispatching thread).
 
-However, if you try to load the contributors via the `BACKGROUND` option, you can see that the list is updated but
-nothing changes.
+However, if you try to load the contributors via the `BACKGROUND` option, you can see that whilethe list is updates,
```
Comment (kochaika) [inline Coroutines/Callbacks/task.md:41]: `whilethe` -- I guess it's a typo

File: Coroutines/Callbacks/task.md [hunk 2]
```diff
@@ -22,10 +22,10 @@ which uses callbacks instead of blocking calls.
 
    ![The freed main thread](images/background.png)
 
-2. The signature of the `loadContributorsBackground()` function changes. It takes an `updateResults()`
-   callback as the last argument to call it after all the loading completes.
-3. Now when the `loadContributorsBackground()` is called, the `updateResults()` call goes in the callback, not immediately
-   afterward as it did before:
+2. The signature of the `loadContributorsBackground()` function has changed. It now takes an `updateResults()`
+   callback as the last argument to be invoked once all the data has been loaded.
+3. Once the `loadContributorsBackground()` is called, the invokation of  `updateResults()` is moved to the callback, instead of being executed immediately
```
Comment (kochaika) [inline Coroutines/Callbacks/task.md:27]: `invokation` -- should it be `invocation`?

**Review by kochaika — CHANGES_REQUESTED** [review]

File: Coroutines/Callbacks/task.md (hunk 1 above)
Comment (stephen-hero) [reply Coroutines/Callbacks/task.md:41]: right.. should be: "while the list updates,"

File: Coroutines/Callbacks/task.md (hunk 2 above)
Comment (stephen-hero) [reply Coroutines/Callbacks/task.md:27]: Absolutely. " Invocation"

**Review by kochaika — APPROVED** [review]

**Pull request merged**